docker-entrypoint/dependencies/entrypoint.go

```diff
@@ -0,0 +1,80 @@
+package entrypoint
+
+import (
+	"github.com/stackanetes/docker-entrypoint/logger"
+	//	"k8s.io/kubernetes/pkg/client/restclient"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"os"
+	"sync"
+	"time"
+)
+
+var (
+	Dependencies     []Resolver      // List containing all dependencies to be resolved
+	DependencyPrefix = "DEPENDENCY_" //Prefix for env variables
+	Namespace        string          //Namespace for containers
```

Comment:
Is there a reason this is a global variable instead of a field of the Entrypoint struct?